src/Database/Expression/Comparison.php

```diff
@@ -1,5 +1,6 @@
 <?php
 declare(strict_types=1);
 
-// @deprecated 4.1.0 Add backwards compatible alias.
-class_alias('Cake\Database\Expression\ComparisonExpression', 'Cake\Database\Expression\Comparison');
+class_exists('Cake\Database\Expression\ComparisonExpression');
+
+deprecationWarning('`Comparison` deprecated in 4.1, use `ComparisonExpression` instead.');
```

Comment:
Wont this be a bit too harsh at this point?

Reply:
Removed, but it's only for users who manually instantiate Comparison objects.